src/EnergyPlus/FluidProperties.cc

```diff
@@ -4867,9 +4847,7 @@ namespace FluidProperties {
         bool NeedOrphanMessage = true;
         int NumUnusedRefrig = 0;
 
-        auto &df = state.dataFluidProps;
-
-        for (auto const *refrig : df->refrigs) {
+        for (auto const *refrig : state.dataFluidProps->refrigs) {
```

Comment:
[src/EnergyPlus/FluidProperties.cc:4870]:(style),[constVariable],Variable 'df' can be declared as reference to const